cluster/common.sh

```diff
@@ -1052,6 +1075,24 @@ function create-kubeconfig() {
   local cluster_args=(
       "--server=${KUBE_SERVER:-https://${KUBE_MASTER_IP}}"
   )
+
```

Comment:
this function is a dup and has been removed. please update the branch.

Reply:
There is a separate PR for that - I just noticed it has still not been merged, that's why the change didn't get picked up. https://github.com/CentaurusInfra/arktos/pull/939
@XiaoningDing Can you please approve and merge

Reply:
fixed it for this PR as well.